web/src/Create.tsx

```diff
@@ -86,6 +93,21 @@ function Create() {
     }
   }, [searchParams]);
 
+    useEffect(() => {
+    const getissuedata = async () => {
+      const issue_path = link?.slice(19)
+      const response = await axios.get(`https://api.github.com/repos/${issue_path}`)
```

Comment:
how does this work for unauthenticated users? Is there any limit on this kind of requests?

Reply:
Right now the requests are made by unauthenticated users, so the limit is 60 requests per hour. This can be increased to 5000 req/hr if we make authenticated requests by using a github account.
Another diversion to this  problem would be using OAuth2.0, using which we can make 5000 req/hr but these would be users' requests not ours.So we can get 5000 req/hr for each user of the app.